server/container_start.go

```diff
@@ -34,8 +39,20 @@ func (s *Server) StartContainer(ctx context.Context, req *pb.StartContainerReque
 		if err := s.ContainerStateToDisk(c); err != nil {
 			log.Warnf(ctx, "unable to write containers %s state to disk: %v", c.ID(), err)
 		}
+
+		if useHighPerformanceRuntimeHandler && shouldCPULoadBalancingBeDisabled(c.Annotations()) {
+			if err := setCPUSLoadBalancing(c, true, schedDomainDir); err != nil {
+				log.Warnf(ctx, "failed enable back the container %q CPUs load balancing: %v", c.ID(), err)
+			}
+		}
 	}()
 
+	if useHighPerformanceRuntimeHandler && shouldCPULoadBalancingBeDisabled(c.Annotations()) {
+		if err := setCPUSLoadBalancing(c, true, schedDomainDir); err != nil {
+			return nil, fmt.Errorf("failed to disable the container %q CPUs load balalancing: %v", c.ID(), err)
```

Comment:
nit 1: the logic here feels backwards. the function is called `setCPUSLoadBalancing` but failure to set to true results in us returning an error that says we `failed to disable the ... load balancing`
nit 2: s/balalancing/balancing/g

Reply:
done